<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReactionMeter</class>
 <widget class="QMainWindow" name="ReactionMeter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ReactionMeter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="0" colspan="4">
       <widget class="QProgressBar" name="progressBar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="setupTimeButton">
        <property name="text">
         <string>Setup time</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QLabel" name="programLabel">
        <property name="text">
         <string>Program:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="importProgramsButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Import programs</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QPushButton" name="exportProgramsButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Export program</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="loadProgramButton">
        <property name="text">
         <string>Load program</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="startButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Start measurement</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2" colspan="2">
       <widget class="QPushButton" name="stopButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop measurement</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QPushButton" name="exportResultsButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Export results</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2" colspan="2">
       <widget class="QPushButton" name="shutdownButton">
        <property name="text">
         <string>Shutdown</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLabel" name="timeLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
